File: fix_admin.py
import os

path = r"c:\Users\Rajeswara\Downloads\EndOfHunger\dineflowreact\src\pages\AdminPage.js"

with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# 1-based line numbers to remove
lines_to_remove = []
lines_to_remove.extend(range(251, 254)) # 251, 252, 253
lines_to_remove.extend(range(350, 353)) # 350, 351, 352
lines_to_remove.extend(range(368, 370)) # 368, 369
lines_to_remove.extend(range(413, 415)) # 413, 414
lines_to_remove.extend(range(428, 430)) # 428, 429
lines_to_remove.extend(range(438, 440)) # 438, 439

new_lines = [line for i, line in enumerate(lines, 1) if i not in lines_to_remove]

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

print(f"Removed {len(lines) - len(new_lines)} lines.")

File: fix_admin.js
const fs = require('fs');
const path = "c:\\Users\\Rajeswara\\Downloads\\EndOfHunger\\dineflowreact\\src\\pages\\AdminPage.js";

try {
    const content = fs.readFileSync(path, 'utf8');
    // Handle both CRLF and LF
    const lines = content.split(/\r?\n/);

    const linesToRemove = new Set();
    const addRange = (start, end) => {
        for (let i = start; i <= end; i++) linesToRemove.add(i);
    };

    addRange(251, 253);
    addRange(350, 352);
    addRange(368, 369);
    addRange(413, 414);
    addRange(428, 429);
    addRange(438, 439);

    const newLines = lines.filter((_, index) => !linesToRemove.has(index + 1));

    // Join with original line ending if possible, or just \n
    fs.writeFileSync(path, newLines.join('\n'), 'utf8');
    console.log(`Removed ${lines.length - newLines.length} lines.`);
} catch (err) {
    console.error(err);
}
